relayer/middleware/metrics.middleware.ts

```diff
@@ -128,7 +148,15 @@ export function metrics<C extends StorageContext>(
 
   const totalDuration = new Histogram({
     name: "vaas_total_duration",
-    help: "Processing time in ms for relaying",
+    help: "Processing time in ms since VAA was published",
+    buckets: opts.processingTimeBuckets ?? defaultTimeBuckets,
```

Comment:
I didn't realise about this on the last PR, but I think we'll need the ability to use different buckets for the different processing times since they might be quite different :(

Reply:
fixed!